cmd/translate/translate.go

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/konveyor/move2kube/internal/common"
+	"github.com/konveyor/move2kube/internal/move2kube"
+	"github.com/konveyor/move2kube/internal/qaengine"
+	"github.com/konveyor/move2kube/types/plan"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type translateFlags struct {
+	ignoreEnv    bool
+	planfile     string
+	outpath      string
+	srcpath      string
+	name         string
+	curate       bool
+	qadisablecli bool
+	qaskip       bool
+	qaport       int
+	qacaches     []string
+}
+
+const (
+	outpathFlag      = "outpath"
+	curateFlag       = "curate"
+	qadisablecliFlag = "qadisablecli"
+	qaskipFlag       = "qaskip"
+	qaportFlag       = "qaport"
+	qacacheFlag      = "qacache"
+
+	nameFlag      = "name"
+	planFlag      = "plan"
+	sourceFlag    = "source"
+	ignoreEnvFlag = "ignoreenv"
+)
```

Comment:
for the variables and values it shares with normal cmd, can we put it as a common one, so that they don't become inconsistent? Similarly for the functions. Only those functions/variables that we do change should be here. We can possible have some common files in cmd folder.

Reply:
you can't import from `main` package so it will have to be a separate folder and package called `common`

Reply:
it need not be named main.